### PR TITLE
clarify that we indeed support volumes, just not for top level keys

### PR DESCRIPTION
**Pull request by whiteotter**
image

### CODE REVIEW

File: _pro/builds-and-configuration/services.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ The following features available in Docker Compose are not available on Codeship
   * `tmpfs`
   * `ulimits`
   * `volume_driver`
-  * `volumes` (top-level key)
+  * `volumes` ([we do support volumes]({% link _pro/builds-and-configuration/docker-volumes.md %}), just not as a [top-level key](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes-volume_driver))
```
Comment (mlocher) [inline _pro/builds-and-configuration/services.md]: Can we reword this as e.g. 

> `volumes` (Compose v2 top level definition. Service specific volumes are supported, see "[Using Volumes](https://documentation.codeship.com/pro/builds-and-configuration/docker-volumes/)")

Not really happy with my wording either, it's more a suggestion/starting point for further iteration.

**Comment by ethangj** [comment]
@whiteotter @mlocher I'd be in favor of just taking volumes out of the list entirely.

**Comment by mlocher** [comment]
That would work for me as well, if we have a way to monitor how many people try to use them via the top level entry (could be as "easy" as looking for tickets)

**Comment by ethangj** [comment]
We ignore if it's in top level and it won't work as the user anticipates, so hopefully they contact us and/or find the volumes doc anyway
